src/main/java/ua/danit/final_project/dto/ShiftCommentDto.java

```diff
@@ -0,0 +1,44 @@
+package ua.danit.final_project.dto;
+
+import lombok.Data;
+import ua.danit.final_project.entities.Employee;
+import ua.danit.final_project.entities.Position;
+import ua.danit.final_project.entities.ShiftComment;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+public class ShiftCommentDto implements Serializable {
+
+  private Long id;
+
+  private String text;
+
+  private Timestamp date;
+
+  private String forename;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "forename" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=92&issues=AWWkvKScfnKVigJDXUg0&open=AWWkvKScfnKVigJDXUg0)

[ISSUE_KEY]: AWWkvKScfnKVigJDXUg0